File: mario.c
#include<cs50.h>

#include<stdio.h>

int main (void)
{
    int n,h,w,i,j;
    //h is height & w is width.
    
    do
    {
        n=get_int("Enter any no. b/w 0 & 8 -> ");
    }
    while(n<=0 || n>8);
    
    for(i=1;i<=n;i++)
    {
       for(h=i;h<n;h++)
       {
           printf(" ");
       }
       for(w=1;w<=i;w++)
       {
           printf("#");
       }
       printf("  ");
       for(j=1;j<w;j++)
       {
        printf("#");   
       }
       printf("\n");
    }
}

File: mario.py
from cs50 import get_int  # To use get_int.
n = 0
while True:
    n = get_int ("Enter the desired height of your pyramid: ")
    if (n > 0 and n < 9):
        break  # To make sure the user enters valid no.
for i in range(n):
    for h in range(n - i - 1):  # To decrease range for each succesive row.
        print(" ", end="")
    for w in range(i + 1):  # To print an increasing no. of # for each succesive row.
        print("#", end="")
    print()  # New Line